docs_app/content/tutorial/6-chaining-operators.md

```diff
@@ -0,0 +1,145 @@
+# 6. Chaining Operators
+
+## TLDR:
+
+- `pipe` is a functional programming concept that executes functions in order, passing the result of each one to the next.
+- Use the `Observable` [`pipe`](API) method to chain operators together and build new Observables.
+- You can create many small chains to shorten larger chains.
+
+## Overview
+
+One of central points of RxJS are combining its many operators in interesting and useful ways. This is done by [chaining](GL) the operators together using functional programming tricks built into RxJS for convenience.
+
+If an operator, as stated in ["what is an operator"](4-what-is-an-operator.md#Operators), is a "thing that can perform an operation", on a collection, then we might be able to go back to our [collection of apples example](4-what-is-an-operator.md#Collections) from that section.
+
+Let's say we had a machine that would sort the apples (bad and good), and we also had a machine that could peel apples, and perhaps a machine that could slice the apples. We could then, in theory, chain them all together with some sort of conveyor and sort, peel, and slice the apples all in one overall operation. In this example we would be [chaining](CL) the operations, and the apples, as we processed them would be [streaming](GL) through those operations.
+
+## Functional Chaining With Pipe
```

Comment:
Great section to clarify what the pipe method basically does. Good explanation and example.